module-4/metascan/README.md

```diff
@@ -0,0 +1,94 @@
+# PDF Metadata Extractor
+
+This Python script downloads PDF files from a specified website and extracts metadata such as title, author, creation date, etc., saving them into a CSV file.
+
+## Features
+
+- **Web Scraping**: Extracts all PDF links from a given website using `BeautifulSoup` and `requests` modules.
+- **PDF Download**: Downloads each PDF file found to a local directory (`pdfs/`).
+- **Metadata Extraction**: Uses the `PyPDF2` library to extract metadata (title, author, creation date, etc.) from each downloaded PDF file.
+- **CSV Export**: Saves extracted metadata into a CSV file with semicolon (`;`) as a delimiter for better readability.
+- **Error Handling**: Provides basic error handling for network requests and PDF metadata extraction failures.
```

Comment:
should not be listed under `Features` since this should be normal implementation that covers errors that could arise during execution and need to be handled correspondingly.